### DISCUSSION TITLE
Rustdesk removed by Webroot endpoint protection as malware

### DISCUSSION THREAD

**Discussion by wonderly2** [rustdesk/rustdesk#10004] [General]
Our Webroot endpoint protection has removed a particular version of Rustdesk from 2 machines. Is there any information on this?

Filename	RUSTDESK.EXE
Pathname	%programfiles%\rustdesk\
Filesize	266.2 KB
MD5	272595DC239C416F97D938EDF06B2FFF
Determination	Bad
Malware Group	Win.Riskware.Rustdesk
First Seen	Nov 20th 2024, 12:20
Last Seen	Nov 20th 2024, 12:20
Vendor	com.carriez
Product	rustdesk
Version	1.2.3+39

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2485
